docs/UserGuide.md

```diff
@@ -290,166 +306,153 @@ The `-c` flag indicates the property's new characteristics.<br>
-The `-pr` flag indicates that we will be sorting buyers according to the specified priority level.<br>
-The `-t` flag indicates that we will be sorting buyers according to entry time.<br>
+The `-n` flag indicates to sort buyers by name in ascending or descending order.<br>
+The `-r` flag indicates to sort buyers by price range in ascending (by lower bound) or descending order (by upper bound).<br>
+The `-pr` flag indicates to sort buyers by priority level in ascending or descending order.<br>
+The `-t` flag indicates to sort buyers by entry time in ascending or descending order.<br>
 
 Examples:<br>
-`sortbuyers -pr DESC`: Sort buyers from `HIGH` priority level to `LOW` priority level.<br>
-`sortbuyers -r ASC` Sort buyers according to the upper price bound in ascending order.<br>
-`sortbuyers -t ASC` Sort buyers according to the time of entry from oldest to most recent.
+`sortbuyers -pr DESC`: Sorts buyers from `HIGH` priority level to `LOW` priority level.<br>
+`sortbuyers -r ASC`: Sorts buyers according to the lower price bound in ascending order.<br>
+`sortbuyers -t ASC`: Sorts buyers according to the time of entry from least recent to most recent.
 
 #### Sort properties in database: `sortprops`
 
-Sorts properties in the database according to a single given condition, and updates the visible property list.
+Sorts properties in the database according to a single given condition.
 
-Syntax: `sortprops {-n PROPERTYNAME <ASC/DESC>} {-p PRICE <ASC/DESC>} {-t ENTRYTIME <ASC/DSC>}`
+Syntax: `sortprops [-n NAME<ASC/DESC>] [-p PRICE<ASC/DESC>] [-t ENTRY TIME<ASC/DESC>]`
 
-The `-n` flag indicates that we will be sorting properties by name in ascending or descending order.<br>
-The `-p` flag indicates that we will be sorting properties by listed price in ascending or descending order.<br>
-The `-t` flag indicates that we will be sorting buyers according to entry time.
+The `-n` flag indicates to sort properties by name in ascending or descending order.<br>
+The `-p` flag indicates to sort properties by price in ascending or descending order.<br>
+The `-t` flag indicates to sort properties by entry time in ascending or descending order.
 
 Examples:<br>
-`sortprops -p DESC`: Sort properties from highest listed price to lowest.<br>
-`sortprops -t ASC` Sort properties according to the time of entry from oldest to most recent.
+`sortprops -p DESC`: Sorts properties from highest to lowest price.<br>
+`sortprops -t ASC`: Sorts properties by time of entry from least recent to most recent.
 
 ### Match Commands
 #### Match specified buyer to properties: `matchbuyer`
 
-Intelligently matches a buyer in the database to properties that fit their criteria.
+Intelligently matches a buyer in the database to all properties that they might be interested in.
+These properties must be within the buyer's price range and satisfy at least one of the buyer's characteristics.
```

Comment:
Perhaps we don't need to include implementation-specific info in the UG?

Reply:
We received a bug report that the criteria of matching isn't clear enough, so I put it in to make it easier for testers to test!